src/components/publiq-ui/Alert.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import { Alert as BootstrapAlert } from 'react-bootstrap';
+
+const AlertVariants = {
+  INFO: 'info',
+  SUCCESS: 'success',
+  DANGER: 'danger',
+  WARNING: 'warning',
+};
+
+const Alert = ({ variant, visible, dismissible, children }) => (
+  <BootstrapAlert variant={variant} hidden={!visible} dismissible={dismissible}>
```

Comment:
Are these components (`Alert`, `Button`) considered "done" or will we still do some changes like make it use a `Box` base component etc? Because I'm curious if we really need to use the Bootstrap components for simple components like the alerts and buttons, especially if we're overriding most colors and other styling anyway. But that's not something we have to do in this PR if these components are just examples to demonstrate the setup.

Reply:
No, far from done, the next PR adds some theming mechanisms, this PR is just mostly testing if it works